src/screens/FeedScreen/ContentPoll.js

```diff
@@ -192,26 +193,26 @@ const stylesComponent = (pollLength) =>
       fontFamily: fonts.inter[400],
       fontSize: normalizeFontSize(24),
       lineHeight: 24,
-      color: COLORS.black
+      color: colors.black
     },
     textComment: {
       fontFamily: fonts.inter[400],
       fontSize: normalizeFontSize(12),
       lineHeight: 18,
-      color: COLORS.gray8
+      color: colors.gray
     },
     usernameComment: {
       fontFamily: fonts.inter[500],
       fontWeight: '900',
       fontSize: normalizeFontSize(12),
       lineHeight: 24,
-      color: COLORS.black
+      color: colors.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.black` to `colors.black`. Make sure that `colors.black` is defined and is the intended color.
```diff
- color: COLORS.black
+ color: colors.black
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->